src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_params.py

```diff
@@ -547,6 +545,11 @@ def load_arguments(self, _):
         c.argument('description', help='the description of the gallery image definition')
     with self.argument_context('sig image-definition update') as c:
         c.ignore('gallery_image')
+
+    with self.argument_context('sig image-version') as c:
+        deprecated_option = c.deprecate(target='--gallery-image-version-name', redirect='--gallery-image-version', hide=True)
```

Comment:
I'd recommend an expiration here of 2.1.0 so that this is slated for removal at the next "breaking release".

Reply:
Agreed. We only have 40 unique subscriptions which have ever used this command. We can just pull the plug in the upcoming one.